bundles/org.openhab.binding.tapocontrol/README.md

```diff
@@ -43,21 +56,32 @@ This is used for device discovery and to create a handshake (cookie) to act with
 
 The thing has the following configuration parameters:
 
-| Parameter          | Description                                                          |
-|--------------------|----------------------------------------------------------------------|
-| username           | Username (eMail) of your Tapo-Cloud                                  |
-| password           | Password of your Tapo-Cloud                                          |
+| Parameter              | Description                                                                                         |
+|------------------------|-----------------------------------------------------------------------------------------------------|
+| username               | Username (eMail) of your Tapo-Cloud                                                                 |
+| password               | Password of your Tapo-Cloud                                                                         |
+| cloudDiscovery         | Use Cloud Discovery-Service to get all in Tapo-App registered devices. Includes DeviceName. IP-Address and Encryption has to set manually                     |
+| udpDiscovery           | Use UDP Discovery-Service to discover online devices in the local network. Includes Encryption and IP-Address. Results will be merged with cloud discovery    |
+| onlyLocalOnlineDevices | [advanced] Uses Cloud and UPD-Discovery to get more informations but will only discover online devices via UDP |
+| broadcastAddress       | [advanced] Set broadcast address to your local subnet  if you have problems with default address   |
+| discoveryInterval      | [advanced] Interval in minutes when a background device scan should be executed. Default is 60     |
```

Comment:
Minor: can you fix the table markdown?

Reply:
Tried to fix it, but because of long text it's not perfect. if you have a better idea please post it :)